apps/prairielearn/src/pages/partials/instructorInfoPanel.ejs

```diff
@@ -83,8 +85,9 @@
         <code><%= JSON.stringify(variant.true_answer); %></code>
     </div>
   </div>
+  <% } %>
 
-  <% if ((typeof question_context !== "undefined") && (question_context == "instructor" || question_context == "manual_grading")) { %>
+  <% if ((typeof variant !== "undefined") && (typeof question_context !== "undefined") && (question_context == "instructor" || question_context == "manual_grading")) { %>
```

Comment:
This hides the issue reporting button/form if a variant wasn't created, because that form relies on a variant ID.